raster/src/main/scala/geotrellis/raster/io/geotiff/MultiBandGeoTiffSubsetMethod.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster.io.geotiff
+
+import geotrellis.raster.{MethodExtensions, Tile, ArrayMultiBandTile}
+import geotrellis.raster.io.geotiff._
+
+
+trait MultiBandGeoTiffSubsetMethod[T <: MultiBandGeoTiff] extends MethodExtensions[T] {
+
+  def permutation(m: Map[Int, Int]): ArrayMultiBandTile = {
```

Comment:
Why do we need a map argument at all? I don't understand where we'd want to call permutation.

Also these methods are core enough to MultiBandTile to have directly in the MultiBandTile trait.